kDrive/AppDelegate.swift

```diff
@@ -107,11 +107,12 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
         Task {
             try! await Task.sleep(nanoseconds:5_000_000_000)
             print("coucou")
-            let somePublicShare = URL(string: "https://kdrive.infomaniak.com/app/share/140946/01953831-16d3-4df6-8b48-33c8001c7981")
-            //await UIApplication.shared.open(somePublicShare!) // opens safari
+            // a public share expired
+            let somePublicShare = URL(string: "https://kdrive.infomaniak.com/app/share/140946/81de098a-3156-4ae6-93df-be7f9ae78ddd")
```

Comment:
We can merge this to the host PR, it does not matter yet. The last PR will do the cleanup.